crates/pallet-subspace/src/lib.rs

```diff
@@ -1193,7 +1194,8 @@ impl<T: Config> Pallet<T> {
             // We assign the maximum priority for any vote.
             .priority(TransactionPriority::MAX)
             // Should be included in the next block or block after that, but not later
-            .longevity(1)
+            .longevity(2)
```

Comment:
Something also seemed to be fixed :D

Reply:
Yes, longevity is a bit confusing when you want to dial it in "just right", from debugging it turned out that longevity needs to be increased to make it into the block `n+2` where `n` is parent block. It would have worked as is, but transactions would be lost quite often.